pom.xml

```diff
@@ -96,7 +97,7 @@
             <dependency>
                 <groupId>com.google.guava</groupId>
                 <artifactId>guava</artifactId>
-                <version>30.1-jre</version>
+                <version>30.1.1-jre</version>
```

Comment:
This is a transitive dep (with a version bump because a vuln IIRC) we might be able to remove it now